monkey/monkey_island/linux/install_mongo.sh

```diff
@@ -1,40 +1,58 @@
 #!/bin/bash
 
-export os_version_monkey=$(cat /etc/issue)
+exists() {
+  command -v "$1" >/dev/null 2>&1
+}
+
+os_version_monkey=$(cat /etc/issue)
+export os_version_monkey
 MONGODB_DIR=$1 # If using deb, this should be: /var/monkey/monkey_island/bin/mongodb
 
-if [[ ${os_version_monkey} == "Ubuntu 16.04"* ]] ;
-then
-	echo Detected Ubuntu 16.04
-	export tgz_url="https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1604-3.6.12.tgz"
-elif [[ ${os_version_monkey} == "Ubuntu 18.04"* ]] ;
-then
-	echo Detected Ubuntu 18.04
-	export tgz_url="https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1804-4.2.0.tgz"
-elif [[ ${os_version_monkey} == "Debian GNU/Linux 8"* ]] ;
-then
-	echo Detected Debian 8
-	export tgz_url="https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-debian81-3.6.12.tgz"
-elif [[ ${os_version_monkey} == "Debian GNU/Linux 9"* ]] ;
-then
-	echo Detected Debian 9
-	export tgz_url="https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-debian92-3.6.12.tgz"
+if [[ ${os_version_monkey} == "Ubuntu 16.04"* ]]; then
+  echo Detected Ubuntu 16.04
+  export tgz_url="https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1604-3.6.12.tgz"
+elif [[ ${os_version_monkey} == "Ubuntu 18.04"* ]]; then
+  echo Detected Ubuntu 18.04
+  export tgz_url="https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1804-4.2.0.tgz"
+elif [[ ${os_version_monkey} == "Debian GNU/Linux 8"* ]]; then
+  echo Detected Debian 8
+  export tgz_url="https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-debian81-3.6.12.tgz"
+elif [[ ${os_version_monkey} == "Debian GNU/Linux 9"* ]]; then
+  echo Detected Debian 9
+  export tgz_url="https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-debian92-3.6.12.tgz"
 else
-	echo Unsupported OS
-	exit -1
+  echo Unsupported OS
+  exit 1
 fi
 
 TEMP_MONGO=$(mktemp -d)
-pushd ${TEMP_MONGO}
-wget ${tgz_url} -O mongodb.tgz
+pushd "${TEMP_MONGO}" || {
+  echo "Pushd failed"
+  exit 1
+}
+
+if exists wget; then
```

Comment:
Notice that mongodb is installed after some wget calls in the deployment scripts, so it would be best to also add the same check to the `deploy_linux.sh`

Reply:
Added